frame/system/src/lib.rs

```diff
@@ -1483,9 +1497,11 @@ impl<T: Trait + Send + Sync> CheckWeight<T> where
 	fn get_priority(info: &DispatchInfoOf<T::Call>) -> TransactionPriority {
 		match info.class {
 			DispatchClass::Normal => info.weight.into(),
-			DispatchClass::Operational => Bounded::max_value(),
+			// Don't use up the whole priority space, to allow things like `tip`
+			// to be taken into account as well.
+			DispatchClass::Operational => TransactionPriority::max_value() / 2,
```

Comment:
how does priority works ? the lowest the first ? if so then are normal transaction more priorities than operational ones ?

Reply:
The other way around. `0` is lowest priority. Here we set to `2**63` so it's quite unlikely that `Normal` transactions will ever reach that, but we can make it a configurable parameter as well if we want to be super precise.

Reply:
ah I got confused by mandatory but now I understand 👍